mvlearn/semi_supervised/ctregression.py

```diff
@@ -0,0 +1,580 @@
+# python version 3.7.6
+
+import pandas as pd
+import numpy as np
+from sklearn.neighbors import KNeighborsRegressor
+import random
+from scipy.spatial.distance import minkowski
+from ..utils.utils import check_Xs, check_Xs_y_nan_allowed
+from .base import BaseCoTrainEstimator
+
+# Implements Multi view Co training Regression for 2 view data
+
+
+class CTRegressor(BaseCoTrainEstimator):
+    r"""
+    This class implements the co-training regression for supervised
+    and semisupervised learning with the framework as described in [#1CTR].
+    The best use case is when nthe 2 views of input data are sufficiently
+    distinct and independent as detailed in [#1CTR]. However this can also
```

Comment:
to render properly, references should be written as "[#1CTR]_" with the trailing underscore